api/server.js

```diff
@@ -44,7 +44,7 @@
      });
  });
  
- const PORT = process.env.PORT || 5000; // Setting the port number for the server
+ const PORT = process.env.PORT || 5001; // Setting the port number for the server
```

Comment:
Why is the port number changed?

Reply:
Something was wrong with my port 5000 when testing. Changed it to 5001 and it worked flawlessly, I can change this back to 5000